mergingArduino/src/mergingArduino/mergingArduino.ino

```diff
@@ -99,6 +100,14 @@ void automaticObstacleAvoidance(){
   }
 }
 
+//code entered here will be executed if the rear sensor is triggered
+void doRear() {
+	if (back.getDistance() < MAX_DISTANCE && back.getDistance() > 0) {
```

Comment:
This line is conducting **two separate** measurements and therefore nothing guarantees that they will be the same number! Please revise.